src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java

```diff
@@ -255,4 +257,38 @@ public void setPriority(String priority) {
     public void setAdditionalProperty(String name, Object value) {
         this.additionalProperties.put(name, value);
     }
+
+    public boolean containsSearchTerm(String searchTerm) {
+        return (StringUtils.containsIgnoreCase(this.columnHeader, searchTerm) ||
+            StringUtils.containsIgnoreCase(this.displayName, searchTerm) ||
+            StringUtils.containsIgnoreCase(this.description, searchTerm));   
+    }
+    
+    public boolean containsAllSearchTerms(List<String> searchTerms) {
+        boolean containsAllSearchTerms = true;
+        for (String searchTerm : searchTerms) {
+            if (!containsSearchTerm(searchTerm)) {
+                containsAllSearchTerms = false;
```

Comment:
maybe simply return `false` here to prevent somehow overriding this variable. and then return `true` below since going through the entire for loop indicates that all the search terms were found.